include/crm_internal.h

```diff
@@ -43,6 +43,37 @@
 #  include <crm/common/xml_internal.h>
 #  include <crm/common/internal.h>
 
+#  include <locale.h>
+#  include <libintl.h>
+#  define PACKAGE_NAME "pacemaker"
+
+#ifdef ENABLE_NLS
+#  define _(x) gettext(x)
+#else
+#  define _(x) x
+#endif
+
+static inline gboolean is_zh_language(void)
+{
+    FILE *fp = NULL;
+    char data[128] = {'0'};
+    char *p = NULL;
+    char* ret = NULL;
+    fp = fopen("/etc/locale.conf", "r");
+    if (fp == NULL){
+        return false;
+    }
+    ret = fgets(data, sizeof(data), fp);
```

Comment:
fclose(fp) should happen right after this (currently there are two returns that don't close it)